app/controllers/base_assets_controller.rb

```diff
@@ -22,6 +22,13 @@ def exclude_blank_redirect_url(params)
     params.reject { |k, v| (k.to_sym == :redirect_url) && v.blank? }
   end
 
+  def handle_empty_access_limited_param(params)
```

Comment:
Minor: I think the `_param` suffix is unnecessary, given `params` are passed in as an argument, and inconsistent with `exclude_blank_redirect_url`. I'd prefer the method name to be `handle_empty_access_limited`.

Reply:
Minor: Possibly rename both these methods as follows:

* `exclude_blank_redirect_url ` -> `normalize_redirect_url`
* `handle_empty_access_limited_param` -> `normalize_access_limited`

Reply:
I've renamed the two methods as you've suggested - in two new commits in this branch.